collectors/COLLECTORS.md

```diff
@@ -501,6 +501,40 @@ the `go.d.plugin`.
 These collectors are developed and maintained by third parties and, unlike the other collectors, are not installed by
 default. To use a third-party collector, visit their GitHub/documentation page and follow their installation procedures.
 
+<details>
+<summary>Typical third party Python collector installation instructions</summary>
+<br>
+In general the below steps should be sufficient to use a third party collector.
+
+1. Download collector code file into folder expected by Netdata.
+2. Download default collector configuration file into folder expected by Netdata.
+3. Edit configuration file from step 2 if required.
+4. Enable collector.
+5. [Restart Netdata](/docs/configure/start-stop-restart.md) 
+
+For example below are the steps to enable the [Python ClickHouse collector](https://github.com/netdata/community/tree/main/collectors/python.d.plugin/clickhouse).
+
+```bash
+# download python collector script to /usr/libexec/netdata/python.d/
+sudo wget https://raw.githubusercontent.com/netdata/community/main/collectors/python.d.plugin/clickhouse/clickhouse.chart.py -O /usr/libexec/netdata/python.d/clickhouse.chart.py
+
+# (optional) download default .conf to /etc/netdata/python.d/
+sudo wget https://raw.githubusercontent.com/netdata/community/main/collectors/python.d.plugin/clickhouse/clickhouse.conf -O /etc/netdata/python.d/clickhouse.conf
+
+# enable collector by adding line a new line with "clickhouse: yes" to /etc/netdata/python.d.conf file
+# this will apend to the file if it already exists or create it if not
+sudo echo "clickhouse: yes" >> /etc/netdata/python.d.conf
+
+# (optional) edit clickhouse.conf if needed
+sudo vi /etc/netdata/python.d/clickhouse.conf
+
+# restart netdata
+sudo systemctl restart netdata
```

Comment:
Maybe link to https://learn.netdata.cloud/docs/configure/start-stop-restart since systemctl may not work on all systems